realm/realm-library/src/androidTest/java/io/realm/RealmConfigurationTest.java

```diff
@@ -159,7 +159,7 @@ public void testVersionLessThanDiscVersionThrows() {
         int[] wrongVersions = new int[] { 0, 1, 41 };
         for (int version : wrongVersions) {
             try {
-                Realm.getInstance(new RealmConfiguration.Builder(getContext()).schemaVersion(version).build());
+                realm = Realm.getInstance(new RealmConfiguration.Builder(getContext()).schemaVersion(version).build());
```

Comment:
Doesn't this generate unused lint warning? But no need to change, just a comments :)